docs/features/pootle_fs_statuses.rst

```diff
@@ -21,38 +23,37 @@ to resolve them:
 
 ``conflict``
   Both the Pootle store and the filesystem file have changed. To resolve this
-  conflict use :djadmin:`merge` to merge the two and manage conflict
-  resolution. Or if you wish to discard one or the other use :option:`fetch
-  --force <fetch --force>` or :option:`add --force <add --force>` to keep the
-  filesystem version or Pootle version respectively.
+  conflict use :djadmin:`resolve` to merge the two and manage conflict
+  resolution. Or if you wish to discard one or the other use :option:`resolve
+  --overwrite`.
```

Comment:
I think we need clarity on how handle each of both scenarios:

> Use :command:`resolve --overwrite --pootle-wins` if you want to keep Pootle version, or :command:`resolve --overwrite` if you want to keep the filesystem version.

Reply:
We already have that elsewhere so I didn't really want to repeat that here again, so I've tried to keep it abstract.